README.rst

```diff
@@ -40,14 +40,59 @@ USAGE
 
 If you want to learn more about coala-bears, its functionality and its usage,
 please take a look at our
-`tutorial <http://coala.readthedocs.org/en/latest/Users/Tutorials/Writing_Bears.html#what-is-a-bear>`__.
+`tutorial <http://coala.readthedocs.org/en/latest/index.html>`__.
 
 ABOUT
 =====
 
-coala-bears is a Python package containing all the bears that is used by coala.
-
-Read more at our `documentation <http://coala.readthedocs.org/en/latest/Users/Tutorials/Writing_Bears.html#guide-to-write-a-bear>`__.
+coala-bears is a Python package containing all the bears that are used by coala.
+It has been split from `coala <https://github.com/coala-analyzer/coala>`_.
+With the initial release, it features 56 bears covering 32 languages.
+You can see all of them `here <https://gist.github.com/Adrianzatreanu/cf2d0c8b2ecd542a4860>`_
```

Comment:
Ah, it's in the "pages" section of the sidebar (We don't seem to be showing
it publically in _Sidebar.md yet)
Link: https://github.com/coala-analyzer/coala/wiki/Bears

On Wed, Mar 30, 2016 at 2:09 PM, Attila <notifications@github.com> wrote:

> In README.rst
> <https://github.com/coala-analyzer/coala-bears/pull/300#discussion_r57853934>
> :
>
> >
> >  ABOUT
> >  =====
> >
> > -coala-bears is a Python package containing all the bears that is used by coala.
> > -
> > -Read more at our `documentation <http://coala.readthedocs.org/en/latest/Users/Tutorials/Writing_Bears.html#guide-to-write-a-bear>`__.
> > +coala-bears is a Python package containing all the bears that are used by coala.
> > +It has been split from `coala <https://github.com/coala-analyzer/coala>`_.
> > +With the initial release, it features 56 bears covering 32 languages.
> > +You can see all of them `here <https://gist.github.com/Adrianzatreanu/cf2d0c8b2ecd542a4860>`_
>
> Though I can't find a link, I think this can be a good idea. Is this wiki
> ready already? Can you, please, provide a link if it is? [image: :smile:]
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/300/files/1c930c1e38103799b7c549fcfe2e29b210db8494#r57853934>
>